ui/app/templates/components/clients/history.hbs

```diff
@@ -212,8 +212,8 @@
         type="button"
         class="button is-primary"
         onclick={{queue
-          (action "handleClientActivityQuery" this.startMonth this.startYear "startTime")
           (action (mut this.isEditStartMonthOpen) false)
+          (action "handleClientActivityQuery" this.startMonth this.startYear "startTime")
```

Comment:
reverse so that the modal closes first then the try happens and the loader is added other wise loader will be showing and the modal will be open.

Reply:
Not a request for change but a suggestion that rather than using the `queue` helper here it might be cleaner to mutate the property value in the `handleClientActivityQuery` instead.